devices/legrand.js

```diff
@@ -15,6 +16,23 @@ const readInitialBatteryState = async (type, data, device) => {
     }
 };
 
+const manufacturerOptions = {manufacturerCode: herdsman.Zcl.ManufacturerCode.VANTAGE, disableDefaultResponse: true};
+
+const tzLocal = {
+    auto_mode: {
+        key: ['auto_mode'],
+        convertSet: async (entity, key, value, meta) => {
+            const mode = {'off': 0x00, 'auto': 0x02, 'on_override': 0x03};
+            const payload = {data: Buffer.from([mode[value]])};
+            await entity.command('manuSpecificLegrandDevices3', 'command0', payload);
+            return {state: {'auto_mode': value}};
+        },
+        convertGet: async (entity, key, meta) => {
+            await entity.read('manuSpecificLegrandDevices3', [0x0000], manufacturerOptions);
```

Comment:
In order for this to work there also needs to be an `fromZigbee` converter which parses the read response of this.

Reply:
Thanks. But this is only required if we can read the auto_mode state from the device? Not so sure if it can be done or not sure how to identify the attributes code.
Effectively, as of  now, your recommendations would be to remove it entirely?

Reply:
yes, remove the convertGet, the new state is already returned by the convertSet

Reply:
Ok, thanks. Why did you remove the function from toZigbee.js and put it instead in Legrand.js?